bazel/scripts/run_sudo_tests.sh

```diff
@@ -0,0 +1,110 @@
+
+#!/usr/bin/env bash
```

Comment:
We should do what `shellcheck` wants here. I would even suggest to fix all the shellcheck warnings in this file, but that's not necessary of course.

Reply:
addressed